src/rules/no-trailing-zeros.ts

```diff
@@ -0,0 +1,54 @@
+import type { AST } from "yaml-eslint-parser";
+import { createRule } from "../utils";
+
+export default createRule("no-trailing-zeros", {
+  meta: {
+    docs: {
+      description: "disallow trailing zeros",
+      categories: ["standard"],
```

Comment:
I don't want to include this rule in the preset because it can change how numeric types are treated by YAML parsers. If users want to use it, add the rule to the config yourself.

```suggestion
      categories: null,
```

By the way, additions to presets are breaking changes, so they are only done in major version.